docs/commands/graph.config-set.md

```diff
@@ -1,5 +1,7 @@
 Set the value of a RedisGraph configuration parameter.
 
+Values set using `GRAPH.CONFIG SET` do not persist after server restart.
```

Comment:
Maybe
Values set using `GRAPH.CONFIG SET` are not persisted
?

Reply:
I'm not sure if you suggest changing 'do no persist' to 'are not persisted' (I just changed it)
or do you recommend removing the 'after server restart'. I try to be extra-clear. I think 'after server restart' may make it clearer for other readers what we mean by persisted here.

Reply:
Yeah, It's kinda hard to put in words,
basically what we want to say is runtime changes to graph's configuration via `GRAPH.CONFIG SET` will not be persisted.

Maybe we can get some help phrasing this from @nermiller ?

Reply:
How's this: Values changed using GRAPH.CONFIG SET during runtime do not persist after server restart.

Reply:
@nermiller Sounds good to me ;-)